src/sqlparser/src/ast/ddl.rs

```diff
@@ -32,6 +32,8 @@ use crate::tokenizer::Token;
 pub enum AlterDatabaseOperation {
     ChangeOwner { new_owner_name: Ident },
     RenameDatabase { database_name: ObjectName },
+    SetBarrierIntervalMs { barrier_interval_ms: Option<u32> },
```

Comment:
Can be merged into a `SetParam`.

Reply:
Maybe we can try only define it as `SetParam { option: String, value: String}` in AST, and later resolve the `option` and `value` string in frontend.